src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * @author leslieharland
+ */
+
+package duke.task;
+
+/**
+ * The Class Deadline.
+ */
+public class Deadline extends Task {
+
+    /**
+     * Instantiates a new deadline.
+     *
+     * @param description the description
+     * @param date        the date
+     */
+    public Deadline(String description, String date) {
+        super(description, date);
+        type = "D";
+    }
+
+    /**
+     * Parses the.
```

Comment:
It would be better if the java comment could explain what the method is parsing.